Sources/VitalHealthKit/HealthKit/HealthKitReads.swift

```diff
@@ -290,9 +296,12 @@ func handleMindfulSessions(
 }
 
 func handleProfile(
-  healthKitStore: HKHealthStore
-) async throws -> ProfilePatch {
-  
+  healthKitStore: HKHealthStore,
+  vitalStorage: VitalHealthKitStorage
+) async throws -> (profilePatch: ProfilePatch?, anchors: [StoredAnchor]) {
+
+  let storage_key = "profile"
```

Comment:
[nit] camel >> snake

Reply:
I usually use camel for keys, but I can change. 👍